engineio/transport/polling/polling.go

```diff
@@ -0,0 +1,20 @@
+package polling
+
+import (
+	"errors"
+)
+
+const (
+	separator    = 0x1e
+	binaryPrefix = 'b'
+)
+
+var (
+	ErrNoEnoughBuf          = errors.New("not enough buf to push back")
+	ErrNoSpace              = errors.New("no enough space to write")
+	ErrPingTimeout          = errors.New("ping timeout")
+	ErrSeparatorInTextFrame = errors.New("should not write 0x1e to text frames")
+	ErrNonCloseFrame        = errors.New("has a non-closed frame")
+
+	_ = binaryPrefix
```

Comment:
why are you did it?

Reply:
Which part? `ErrSeparatorInTextFrame`?

Reply:
`_ = binaryPrefix` - is it construction new for me. what is it mean?

Reply:
It's to avoid lint error. binaryPrefix will be used in polling implementation.